allennlp/models/simple_tagger.py

```diff
@@ -54,14 +54,19 @@ def __init__(self,
         self.sequence_loss = torch.nn.CrossEntropyLoss()
 
     # pylint: disable=arguments-differ
-    def forward(self,  # type: ignore
-                tokens: torch.LongTensor,
+    def forward(self,
+                tokens: Dict[str, torch.LongTensor],
                 tags: torch.LongTensor = None) -> Dict[str, torch.Tensor]:
         """
         Parameters
         ----------
-        tokens : torch.LongTensor, required
-            A torch tensor representing the indices of the
+        tokens : Dict[str, torch.LongTensor], required
+            A dictionary of namespaces which have been indexed to their corresponding tensors.
```

Comment:
I would probably start this comment with `The output of TextField.as_array(), which should typically be passed to a TokenEmbedder`, or something similar to that.